test_runner/src/main/kotlin/ftl/gc/GcAuth.kt

```diff
@@ -0,0 +1,76 @@
+package ftl.gc
+
+import com.google.api.client.auth.oauth2.Credential
+import com.google.api.client.extensions.java6.auth.oauth2.AuthorizationCodeInstalledApp
+import com.google.api.client.extensions.jetty.auth.oauth2.LocalServerReceiver
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport
+import com.google.api.client.http.HttpTransport
+import com.google.api.client.json.jackson2.JacksonFactory
+import com.google.api.client.util.store.FileDataStoreFactory
+import ftl.config.FtlConstants
+import java.io.File
+import java.io.IOException
+
+// https://github.com/googleapis/google-oauth-java-client
+// Code from: https://developers.google.com/sheets/api/quickstart/java
+object GcAuth {
+
+    private val HOME = System.getProperty("user.home")
+    private val JSON_FACTORY = JacksonFactory.getDefaultInstance()
+    private var HTTP_TRANSPORT: HttpTransport? = null
+    private var DATA_STORE_FACTORY: FileDataStoreFactory? = null
+
+    private val CRED_FOLDER = File(HOME, ".flank/")
+    val CRED = File(CRED_FOLDER, "StoredCredential")
+
+    // https://github.com/bootstraponline/gcloud_cli/blob/40521a6e297830b9f652a9ab4d8002e309b4353a/google-cloud-sdk/platform/gsutil/gslib/utils/system_util.py#L177
+    private val clientId = "32555940559.apps.googleusercontent.com"
+    private val clientSecret = "ZmssLNjJy2998hD4CTg2ejr2"
+
+    init {
+        try {
+            HTTP_TRANSPORT = GoogleNetHttpTransport.newTrustedTransport()
+            DATA_STORE_FACTORY = FileDataStoreFactory(CRED_FOLDER)
+        } catch (t: Throwable) {
+            t.printStackTrace()
```

Comment:
Not sure if this is copied code or not, but why not let the exception propagate or just rethrow it as a RuntimeException?

Reply:
That's a good point. Updated.